### PR TITLE
feat: onscroll handle notebooks presets

### PR DESCRIPTION
**Pull request by ShmuelLotman**
Closes #

Makes the scroll all purdy for the notebooks presets and removes the flag so that this can be a first class feature

### CODE REVIEW

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 57 | :red_circle:
accessibility | 96 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 100 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=2394)

File: cypress/e2e/shared/flows.test.ts [hunk 1]
```diff
@@ -295,7 +295,7 @@ describe('Flows', () => {
     cy.getByTestID('nav-item-flows').click()
     cy.reload()
     cy.getByTestID('tree-nav').should('be.visible')
-    cy.getByTestID('resource-editable-name').should('be.visible')
+    cy.getByTestID('resource-editable-name').should('exist')
```
Comment (genehynson) [inline cypress/e2e/shared/flows.test.ts:298]: why was this changed required?

Comment (ShmuelLotman) [reply cypress/e2e/shared/flows.test.ts:298]: because in the slimmed down cypress window its not visible due to the box of the presets, so we need it to _be_ there,but being visible wasn't possible

File: src/flows/components/FlowsIndex.tsx [hunk 2]
```diff
@@ -61,32 +107,62 @@ const FlowsIndex = () => {
...
     >
-      <PageHeader fullWidth={false}>
+      <PageHeader
+        fullWidth={false}
+        className={`${showButtonMode && 'withButtonHeader marginHeader'}`}
+      >
         <Page.Title title={PROJECT_NAME_PLURAL} />
+        {showButtonMode && (
+          <>
+            <PresetFlows buttonMode={showButtonMode} />
```
Comment (drdelambre) [inline src/flows/components/FlowsIndex.tsx:117]: why are the concept of "buttonMode" bleeding between these components? since it's already in a boolean conditional, just conditionally render something else

File: src/flows/components/FlowsIndex.tsx [hunk 3]
```diff
@@ -61,32 +107,62 @@ const FlowsIndex = () => {
...
-          {!isFlagEnabled('presetFlows') && <FlowCreateButton />}
-        </Page.ControlBarRight>
-      </Page.ControlBar>
-      <Page.Contents scrollable={true}>
-        {isFlagEnabled('presetFlows') && <PresetFlows />}
-        <FlowCards flows={flowList} search={search} />
-      </Page.Contents>
+      <DapperScrollbars onScroll={scrollHandler} id="scrollFlows">
+        <Page.Contents fullWidth={false} id="fadebox" ref={fadingBoxRef}>
+          <PresetFlows buttonMode={false} />
```
Comment (drdelambre) [inline src/flows/components/FlowsIndex.tsx:142]: surprised this didn't trip up the linter. remove this prop

File: src/flows/components/FlowsIndex.tsx [hunk 4]
```diff
@@ -61,32 +107,62 @@ const FlowsIndex = () => {
...
+          <Page.ControlBar fullWidth={false}>
```
Comment (drdelambre) [inline src/flows/components/FlowsIndex.tsx:143]: is fullWidth explicitly true? if not, remove this one as well

File: src/flows/components/PresetFlows.tsx [hunk 5]
```diff
@@ -17,17 +17,53 @@ const PresetMap = {
...
+  ) : (
     <Grid>
       <Grid.Row>
         <Grid.Column
           widthXS={Columns.Twelve}
           widthSM={Columns.Eight}
           widthMD={Columns.Ten}
         >
-          <div className="flows-index--presetContainer">
+          <div id="presetContainer" className="flows-index--presetContainer">
```
Comment (drdelambre) [inline src/flows/components/PresetFlows.tsx:66]: dont use ids in react, it causes weird shit to happen. if you need to access something directly, use a ref

File: src/flows/components/FlowsIndex.tsx [hunk 6]
```diff
@@ -61,32 +107,62 @@ const FlowsIndex = () => {
       titleTag={pageTitleSuffixer([PROJECT_NAME_PLURAL])}
       testID="flows-index"
     >
-      <PageHeader fullWidth={false}>
+      <PageHeader
+        fullWidth={false}
+        className={`${showButtonMode && 'withButtonHeader marginHeader'}`}
```
Comment (drdelambre) [inline src/flows/components/FlowsIndex.tsx:112]: i didn't realize you created both of these classes. merge their definitions and get rid of one

**Review by drdelambre — CHANGES_REQUESTED** [review]
keep it in a flag until it's live

File: cypress/e2e/shared/flows.test.ts (hunk 1 above)
Comment (genehynson) [reply cypress/e2e/shared/flows.test.ts:298]: how does the next line work then? Cypress' `.click()` only works if the element is visible

Comment (ShmuelLotman) [reply cypress/e2e/shared/flows.test.ts:298]: it works even if its off screen, as far as i could see and local testing bore out that its ok

Comment (genehynson) [reply cypress/e2e/shared/flows.test.ts:298]: huh ok! cypress must have changed that behavior then

File: src/flows/components/FlowsIndex.tsx (hunk 4 above)
Comment (ShmuelLotman) [reply src/flows/components/FlowsIndex.tsx:143]: yeah it looks like it is unfortunately

File: src/flows/components/PresetFlows.tsx (hunk 5 above)
Comment (ShmuelLotman) [reply src/flows/components/PresetFlows.tsx:66]: this was a floating id, thanks for pointing this out, removing it. It wasn't actually doing anything

File: src/flows/components/controlSearchBar.scss [hunk 7]
```diff
@@ -0,0 +1,40 @@
+.cf-page-control-bar--fixed {
+  padding: 0px !important;
+}
+
+.flows-index--presetButtons {
+  animation: fadeIn 0.5s;
+  -webkit-animation: fadeIn 0.5s;
```
Comment (drdelambre) [inline src/flows/components/controlSearchBar.scss:7]: i dont think you need these browser specific rules as the browser wars are over and any rubble that remains should be added by our css parser.

**Review by drdelambre — APPROVED** [review]
